LoRaEngine/modules/LoRaBasicsStationModule/station.conf

```diff
@@ -20,6 +20,8 @@
         /* chan_multiSF_X, chan_Lora_std, chan_FSK provided by LNS */
     },
     "station_conf": {
+        "placeholder": "placeholder",
```

Comment:
for future extensibility, should we rename this placeholder something like routerIdPlaceholder?